library/include/ck/library/host_tensor/device.hpp

```diff
@@ -1,13 +1,47 @@
 #ifndef DEVICE_HPP
 #define DEVICE_HPP
 
+#include "ck/options.hpp"
```

Comment:
Does this include have to be before system headers?

Reply:
Fixed